speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/ConfigEnums.h

```diff
@@ -27,6 +27,21 @@ typedef unsigned int InterventionType;
 
 #define NUM_INTERVENTION_TYPES 4
 
+/// @brief the different ways a race can end
+typedef unsigned int RaceEndType;
+
+#define NORMAL_EXIT   0
+#define RACE_RESTART  1
+#define RACE_EXIT     2
+#define RACE_ABORT    3
+#define RACE_FINISHED 4
+
+/// @brief the different string names of the buttons
+#define PRM_YESSAVE_BUTTON   "yessave"
+#define PRM_IMSURE_BUTTON    "imsure"
+#define PRM_DONTSAVE_BUTTON  "dontsave"
+#define PRM_DONTDELTE_BUTTON "waitdontdelete"
```

Comment:
These are not enums, these are specific to some menu's xml files, move them to the corresponding menu's .cpp files.